### ISSUE TITLE
[10.0] Update Languages UnicodeDecodeError

### ISSUE THREAD

**Issue by thanglt0209** [odoo/odoo#25589]
Impacted versions:
Odoo 10.0-20180214 （Community）
Odoo 10.0+e-20180417 (Enterprise Edition)
Steps to reproduce:
Update Languages: Setting/Languages -> update
![image](https://user-images.githubusercontent.com/10916115/42253822-8810760e-7f7e-11e8-824f-0cdfe1e9dea0.png)
![image](https://user-images.githubusercontent.com/10916115/42253826-926fde1e-7f7e-11e8-8451-0ee7b891a320.png)

Current behavior:
![image](https://user-images.githubusercontent.com/10916115/42253848-acbe54d0-7f7e-11e8-91ed-feaecf2a3687.png)

I had to use Docker to build Odoo (https://github.com/odoo/docker)
```
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/odoo/http.py", line 641, in _handle_exception
    return super(JsonRequest, self)._handle_exception(exception)
  File "/usr/lib/python2.7/dist-packages/odoo/http.py", line 683, in dispatch
    result = self._call_function(**self.params)
  File "/usr/lib/python2.7/dist-packages/odoo/http.py", line 333, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/odoo/service/model.py", line 101, in wrapper
    return f(dbname, *args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/odoo/http.py", line 326, in checked_call
    result = self.endpoint(*a, **kw)
  File "/usr/lib/python2.7/dist-packages/odoo/http.py", line 941, in __call__
    return self.method(*args, **kw)
  File "/usr/lib/python2.7/dist-packages/odoo/http.py", line 506, in response_wrap
    response = f(*args, **kw)
  File "/usr/lib/python2.7/dist-packages/odoo/addons/web/controllers/main.py", line 894, in call_button
    action = self._call_kw(model, method, args, {})
  File "/usr/lib/python2.7/dist-packages/odoo/addons/web/controllers/main.py", line 882, in _call_kw
    return call_kw(request.env[model], method, args, kwargs)
  File "/usr/lib/python2.7/dist-packages/odoo/api.py", line 689, in call_kw
    return call_kw_multi(method, model, args, kwargs)
  File "/usr/lib/python2.7/dist-packages/odoo/api.py", line 680, in call_kw_multi
    result = method(recs, *args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/odoo/addons/base/module/wizard/base_update_translations.py", line 41, in act_update
    tools.trans_export(this.lang, ['all'], buf, 'csv', self._cr)
  File "/usr/lib/python2.7/dist-packages/odoo/tools/translate.py", line 693, in trans_export
    translations = trans_generate(lang, modules, cr)
  File "/usr/lib/python2.7/dist-packages/odoo/tools/translate.py", line 975, in trans_generate
    for root, dummy, files in walksymlinks(path):
  File "/usr/lib/python2.7/dist-packages/odoo/tools/osutil.py", line 38, in walksymlinks
    for dirpath, dirnames, filenames in os.walk(top, topdown, onerror):
  File "/usr/lib/python2.7/os.py", line 296, in walk
    for x in walk(new_path, topdown, onerror, followlinks):
  File "/usr/lib/python2.7/os.py", line 296, in walk
    for x in walk(new_path, topdown, onerror, followlinks):
  File "/usr/lib/python2.7/os.py", line 296, in walk
    for x in walk(new_path, topdown, onerror, followlinks):
  File "/usr/lib/python2.7/os.py", line 286, in walk
    if isdir(join(top, name)):
  File "/usr/lib/python2.7/posixpath.py", line 80, in join
    path += '/' + b
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe6 in position 1: ordinal not in range(128)
```

**Comment by pedrobaeza** [comment]
Are you able to reproduce the problem in runbot https://runbot.odoo.com?

**Comment by mart-e** [comment]
Do you happen to have files with name in unicode?
Also, always try on an up to date version of odoo (or, as Pedro suggested, on a [runbot](http://runbot.odoo.com/))

**Comment by thanglt0209** [comment]
I think the problem is the server environment in Docker.
It had build container by Odoo Docker file. So maybe no problem on runboot.
"""
FROM debian:jessie
MAINTAINER Odoo S.A. <info@odoo.com>

# Install some deps, lessc and less-plugin-clean-css, and wkhtmltopdf
RUN set -x; \
        apt-get update \
        && apt-get install -y --no-install-recommends \
            ca-certificates \
            curl \
            node-less \
            python-gevent \
            python-pip \
            python-renderpm \
            python-support \
            python-watchdog \
        && curl -o wkhtmltox.deb -SL http://nightly.odoo.com/extra/wkhtmltox-0.12.1.2_linux-jessie-amd64.deb \
        && echo '40e8b906de658a2221b15e4e8cd82565a47d7ee8 wkhtmltox.deb' | sha1sum -c - \
        && dpkg --force-depends -i wkhtmltox.deb \
        && apt-get -y install -f --no-install-recommends \
        && apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false -o APT::AutoRemove::SuggestsImportant=false npm \
        && rm -rf /var/lib/apt/lists/* wkhtmltox.deb \
        && pip install psycogreen==1.0

# Install Odoo
ENV ODOO_VERSION 10.0
ENV ODOO_RELEASE 20180214
RUN set -x; \
        curl -o odoo.deb -SL http://nightly.odoo.com/${ODOO_VERSION}/nightly/deb/odoo_${ODOO_VERSION}.${ODOO_RELEASE}_all.deb \
        && dpkg --force-depends -i odoo.deb \
        && apt-get update \
        && apt-get -y install -f --no-install-recommends \
        && rm -rf /var/lib/apt/lists/* odoo.deb
#Install Japanese Font
RUN apt-get update \
    && yes | apt-get install fonts-takao-mincho fonts-takao \
    && fc-cache -f -v

# Copy entrypoint script and Odoo configuration file
COPY ./entrypoint.sh /
COPY ./odoo.conf /etc/odoo/
RUN chown odoo /etc/odoo/odoo.conf

# Mount /var/lib/odoo to allow restoring filestore and /mnt/extra-addons for users addons
RUN mkdir -p /mnt/extra-addons \
        && chown -R odoo /mnt/extra-addons
VOLUME ["/var/lib/odoo", "/mnt/extra-addons"]

# Expose Odoo services
EXPOSE 8069 8071

# Set the default config file
ENV ODOO_RC /etc/odoo/odoo.conf

# Set default user when running the container
USER odoo

ENTRYPOINT ["/entrypoint.sh"]
CMD ["odoo"]
"""

**Comment by mart-e** [comment]
@thanglt0209 still was not able to reproduce it on a docker install. Do you have specific module installed? Are you using unicode somewhere?

> ENV ODOO_RELEASE 20180214

You have customized the docker file. Did you make some other modifications?

In your report you mention enterprise but it seems you are only using the community here.

Also, in your screenshot you are not updating the translations, you are synchronising the terms (which probably does not what you want). Use the public "Load Translation" menu to refresh your translations.

**Comment by thanglt0209** [comment]
@mart-e I'm sorry because of reply late.
I had to try to create the new database just install CRM module and export translation. But I get the same problem. I think the problem is the docker environment. But I don't have any idea about this problem. Have you some idea about that? 
I had installed install xlrd (pip install xlrd), pysftp by (wget https://pypi.python.org/packages/36/60/45f30390a38b1f92e0a8cf4de178cd7c2bc3f874c85430e40ccf99df8fe7/pysftp-0.2.9.tar.gz)  and libffi-dev. Just this.

Thank you very much.

![image](https://user-images.githubusercontent.com/10916115/42798366-74b83d58-89ce-11e8-8bb9-b0be6b77badc.png)
![image](https://user-images.githubusercontent.com/10916115/42798384-830bca1e-89ce-11e8-8656-4daeb8497d99.png)
